containers/vuejs/src/components/PongCanvas.vue

```diff
@@ -65,7 +88,7 @@ const render = (data: {
 }
 
 onMounted(() => {
-  window.addEventListener('resize', drawCanvas) //TODO: replace with render to redraw all objects with correct size
+  window.addEventListener('resize', drawCanvas) // TODO: replace with render to redraw all objects with correct size
```

Comment:
The problem here is that if the game isn't running, the ball and paddles shouldn't be drawn, and we send data a lot so it seems that it isn't necessary. I presume that if you resize the window while  playing that this funciton is called first and then the render funciton is called. Anyhow, I didn't see any issues.